opentech/apply/funds/models.py

```diff
@@ -143,12 +158,60 @@ def process_form_submission(self, form):
                 response = cleaned_data.pop(field.id)
                 cleaned_data[field.block.name] = response
 
+        if cleaned_data.get('email'):
```

Comment:
If we are here then the required field "email" will exist so we don't need to check if an email is provided. 

I think the rest of this section should be a new method `create_user`.